relays/table_transform.ts

```diff
@@ -23,10 +24,10 @@ export default function(msg: Message): void|Message|Message[]|Promise<Message> {
 	let rounds = roundsArray(msg.data.stage.rounds);
 
 	let data = {
-		header: [NUMBER, NAME, TOP].concat(rounds),
-		data: msg.data.ranking.map(function(rank) {
+		header: [RANK, NUMBER, NAME, TOP].concat(rounds),
+		data: msg.data.ranking.map(function(rank, index) {

```

Comment:
Hi Martin, thank you for the quick response!

The goal here was to make it like it was in the old program, So that when two teams have the same rank they would be one after another in a random order.
But I do agree that this is in the scope of responsibility of the fllscoring, so I'll use the rank that comes with the message.

I'm not sure however about showing every rank only once in the display system, it kind of breaks generics. I see two options to go with this:
1. Show it like you did in your first example
2. Give the table module in the displaySystem an ability to accept null values and set them to different values according to a column settings called "default" (some columns would have a default constant value, like the rounds columns, while this column whould have a "repeat" type which will tell him to repeat the last value).